backend/src/openarchiefbeheer/zaken/utils.py

```diff
@@ -92,18 +82,22 @@ def retrieve_zaaktypen_choices() -> list[DropDownChoice]:
         response.raise_for_status()
         data_iterator = pagination_helper(ztc_client, response.json())
 
-    zaaktypen = []
+    zaaktypen = defaultdict(list)
```

Comment:
Given that this will only limit the zaaktypen DB wide, not per view. this seems ok. But do we need to think about having a limited set of zaaktypen for dataset that's currently active (e.g. a filtered set of zaken)? Or zaken on an existing destruction list?

Reply:
In my head the filters were only active when we are in "edit mode", but i see that this is not the case. We could change the zaaktype-choice endpoint to take a destruction list as query param and then filter based on the zaaktypen of the zaken in the destruction list :thinking: If no query param is given, then the filter is DB wide.

What do you think?

Reply:
That would sort of work I guess, it wouldn't filter based on the other filter though, but thats not the case for the other selects either.